src/openMVG/exif/sensor_width_database/sensorDatabaseSort.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+# command line
+parser = argparse.ArgumentParser(description='Sort sensor width camera database')
```

Comment:
not really clear what it does... "Generate a copy of the sensor database sorted by increasing(?) sensor width"?